adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -613,39 +635,38 @@ private boolean verifyAuthenticator(final AccountManager am) {
      */
     @Override
     public UserInfo[] getBrokerUsers() throws OperationCanceledException, AuthenticatorException, IOException {
-
+        final String methodName = ":getBrokerUsers";
         // Calling this on main thread will cause exception since this is
         // waiting on AccountManagerFuture
         if (Looper.myLooper() == Looper.getMainLooper()) {
             throw new IllegalArgumentException("Calling getBrokerUsers on main thread");
         }
 
-        Account[] accountList = mAcctManager.getAccountsByType(AuthenticationConstants.Broker.BROKER_ACCOUNT_TYPE);
-        Bundle bundle = new Bundle();
+        final Account[] accountList = mAcctManager.getAccountsByType(AuthenticationConstants.Broker.BROKER_ACCOUNT_TYPE);
```

Comment:
bundle is used to tell account manager updateCredential call that "I want to get all the users in the broker " back. Since there is no user in the account list, the loop won't even start, and why it's wrong?